common/definition/indexedKeys.go

```diff
@@ -51,6 +51,8 @@ const (
 	CustomBoolField       = "CustomBoolField"
 	CustomDatetimeField   = "CustomDatetimeField"
 	TemporalChangeVersion = "TemporalChangeVersion"
+	CustomNamespace       = "CustomNamespace"
```

Comment:
Can you also update the code to use these definitions where we emit Searchable attributes?

Reply:
done